docs/en/05_Contributing/01_Code.md

```diff
@@ -262,6 +262,38 @@ Helpful hint: You can always edit your last commit message by using:
 
 	$ git commit --amend
 
+## Working with client-side dependencies
+
+From time to time client-side dependencies like jQuery need to be upgraded, added, or removed. We have some tools for dealing with that.
```

Comment:
Make it clearer that this relates to core dependencies, and that users are free to use their own separate library versions in their own build systems.